### PR TITLE
Fix/list styling

### LINKED ISSUE #507: [BUG] List with header and sections - rounded corners (web)
**Describe the bug**
If you have a list with list-header and sections the current styles wont make the first list item with rounded corners.

see screenshot

**Expected behavior**
The first item should have rounded corners even though it has a list header. Current styles of the list on web should just be modified. Right now it's actually excluded with :not(.has-header) which is wrong

**Screenshots**
image

**Additional context**
This only affects web and is caused by the ionic change #102 where the styles and implementation of the list was changed

### PR DESCRIPTION
**Pull request by jakobe**
Fixes #507

### CODE REVIEW

File: src/kirby/components/list/list.component.scss [hunk 1]
```diff
@@ -3,61 +3,90 @@
...
 
-  ion-item {
-    background-color: get-color('white');
-    &:not(:last-of-type).has-divider {
-      border-bottom: 1px solid $divider-color;
-    }
+  &.selected ::ng-deep * {
+    font-weight: font-weight('bold');
+  }
+  &:focus {
```
Comment (jonasjensen-brolstark) [inline src/kirby/components/list/list.component.scss]: There is something wrong with the focus outline:

image

And is it on on purpose outline is always shown and not just when the item is selectable?

File: src/kirby/components/list/list.component.scss [hunk 2]
```diff
@@ -3,61 +3,90 @@
...
-    &.selectable {
-      cursor: pointer;
-      &:hover {
-        background-color: $background-color-hover;
-      }
-    }
+  $list-colors: ('light'); // add supported list item theme colors here
+  @each $color-name, $color-value in $list-colors {
+     &.#{$color-name} {
+             background-color: #{get-color($color-name)};
```
Comment (jonasjensen-brolstark) [inline src/kirby/components/list/list.component.scss]: There is some indentation here that is off.

File: src/kirby/components/list/list.component.scss [hunk 3]
```diff
@@ -3,61 +3,90 @@
...
+}
+
+.footer {
+  background-color: get-color('white');
+  border-top: 1px solid $divider-color;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+
```
Comment (jonasjensen-brolstark) [inline src/kirby/components/list/list.component.scss]: 🔫 pew pew, whitespace. (🔫   ⬅️  my whitespace gun)

File: src/kirby/components/list/list.component.scss (hunk 1 above)
Comment (jakobe) [reply src/kirby/components/list/list.component.scss]: Don't think this is related to the PR...

File: src/kirby/components/list/list.component.scss (hunk 2 above)
Comment (jakobe) [reply src/kirby/components/list/list.component.scss]: Not related to this PR, but fixed :)

**Pull request merged**